routes/api.js

```diff
@@ -1,22 +1,62 @@
 const express = require( "express" );
 const router = express.Router();
+const bcrypt = require( "bcrypt" );
+const passport = require( "passport" );
+
+const passport_config = require( "../util/passportUtil" );
+const { checkNotAuthenticated } = passport_config;
+
+router.post( "/login", passport.authenticate( "local", {
+  successRedirect: "/home",
```

Comment:
Can be addressed in next PR, but you'd want to redirect to /app.

Reply:
Yes, in general I agree. But /app currently has no feature to log out jet. It will be addressed in PR regarding session-handling.